misc/READMORE.md

```diff
@@ -196,7 +196,7 @@ App.listen(port, [](auto *listenSocket) {
 })
 ```
 
-Cancelling listenning is done with the uSockets function call `us_listen_socket_close`.
+Cancelling listening is done with the uSockets function call `us_listen_socket_close`.
```

Comment:
Should `listening` be changed to `listener`?

Reply:
Listening is fine but Cancelling should be Canceling in the American English form, which we all know is the more correct form, less useless letters. Also you missed "toghether" on line 209 should be together